netbout-web/src/test/java/com/netbout/rest/bout/TkIndexTest.java

```diff
@@ -87,5 +89,47 @@ public void rendersBoutPage() throws Exception {
             )
         );
     }
+    /**
+     * TkIndex can search bout messages.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void searchesMessages() throws Exception {
+        final MkBase base = new MkBase();
+        final String urn = "urn:test:99";
+        final User user = base.user(new URN(urn));
+        user.aliases().add("test-search-user");
+        final Alias alias = user.aliases().iterate().iterator().next();
+        final Bout bout = alias.inbox().bout(alias.inbox().start());
+        bout.messages().post("test1");
+        bout.messages().post("test2");
+        bout.messages().post("fest");
+        bout.friends().invite(alias.name());
+        MatcherAssert.assertThat(
+            XhtmlMatchers.xhtml(
+                new RsPrint(
+                    // @checkstyle MultipleStringLiteralsCheck (1 line)
+                    new FkBout(".*", new TkIndex(base)).route(
```

Comment:
@erimerturk let's extract this shares string into a private static final variable since it pertains to multiple test methods.